opencga-client/src/main/python/pyopencga/opencga_client.py

```diff
@@ -130,23 +130,27 @@ def logout(self):
         for client in self.clients:
             client.token = self.token
 
-    def wait_for_job(self, response=None, study=None, job_id=None, retry_seconds=10):
+    def wait_for_job(self, response=None, study_id=None, job_id=None,
+                     retry_seconds=10):
         if response is not None:
-            study = response['studyUuid']
-            job_id = response['uuid']
+            study_id = response['result'][0]['study']['id']
+            job_id = response['result'][0]['uuid']
 
-        if study is None or job_id is None:
-            raise ValueError('Argument "response" or arguments "study" and "job_id" must be provided')
+        if response is None and (study_id is None or job_id is None):
+            raise ValueError('Argument "response" or arguments "study" and'
+                             ' "job_id" must be provided')
 
         if len(job_id.split(',')) > 1:
             raise ValueError('Only one job ID is allowed')
 
         retry_seconds = retry_seconds if retry_seconds >= 10 else 10
         while True:
-            job_info = self.jobs.info(study=study, jobs=job_id, include='status').get_result(0)
+            job_info = self.jobs.info(study=study_id, jobs=job_id,
+                                      include='status').get_result(0)
```

Comment:
I think status is now in _internal_ field. @pfurio can you confirm this?
```suggestion
                                      include='internal.status').get_result(0)
```